src/constants/@modules.d.ts

```diff
@@ -0,0 +1 @@
+declare module "@ledgerhq/hw-app-str";
```

Comment:
@quietbits I'm not sure if this is the best way to handle a 3rd party package with no types package. In js-wallets-sdk we do something similar

https://github.com/stellar/js-stellar-wallets/blob/0425424bba1994944757b24e3aa7d70fdb8f7441/src/types/modules.d.ts#L1